core/models/msgs.go

```diff
@@ -412,8 +377,8 @@ func buildMsgMetadata(m *flows.MsgOut) map[string]interface{} {
 	return metadata
 }
 
-// NewIncomingMsg creates a new incoming message for the passed in text and attachment
-func NewIncomingMsg(cfg *runtime.Config, orgID OrgID, channel *Channel, contactID ContactID, in *flows.MsgIn, createdOn time.Time) *Msg {
+// NewIncomingSurveyorMsg creates a new incoming message for the passed in text and attachment
+func NewIncomingSurveyorMsg(cfg *runtime.Config, orgID OrgID, channel *Channel, contactID ContactID, in *flows.MsgIn, createdOn time.Time) *Msg {
```

Comment:
surveyor messages should be using text type as well I think

https://github.com/nyaruka/mailroom/pull/40/files#diff-96ebd6741cc8a5d068db3fd9ae411d261c2fd36517c2da207fd8d44846f8ae5cR393

Reply:
we can't change msg_type for incoming messages until we update the RP views for Inbox and Flows, so for now I'm just changing it for outgoing.